doc/EnvironmentVariables.md

```diff
@@ -37,8 +37,8 @@
 ## EdgeHub
 | Variable                                | Description                                                                                                                                        | Values                                                                | Default                       |
 |-----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|-------------------------------|
-| amqpSettings:enabled                    | Whether the AMQP protocol head should be enabled                                                                                                   | bool                                                                  | true                          |
-| amqpSettings:port                       | The port for the AMQP protocol head to listen on                                                                                                   | int32                                                                 | 5671                          |
+| amqpSettings__enabled                    | Whether the AMQP protocol head should be enabled                                                                                                   | bool                                                                  | true                          |
```

Comment:
Well, I'm confused here. In the description, it is said PascalCase is proposed, but changes in the doc do not reflect this statement.

shouldn't it be like
```suggestion
| AmqpSettings__Enabled                    | Whether the AMQP protocol head should be enabled                                                                                                   | bool                                                                  | true                          |
```

Reply:
you're right, it should be fixed as well.